Gems/Atom/Feature/Common/Code/Include/Atom/Feature/AuxGeom/AuxGeomFeatureProcessor.h

```diff
@@ -38,7 +38,8 @@ namespace AZ
             // RPI::FeatureProcessor
             void Activate() override;
             void Deactivate() override;
-            void Render(const FeatureProcessor::RenderPacket& fpPacket) override;
+            void Render(const FeatureProcessor::RenderPacket& fpPacket) override;            
```

Comment:
nitpick: extra whitespace was added to this line
```suggestion
            void Render(const FeatureProcessor::RenderPacket& fpPacket) override;
```